README.md

```diff
@@ -69,13 +69,9 @@ algorithm:
    section further below.
 2. Reset the staging (a.k.a. "auto") branch to the PR staging commit.
    The previous state of the staging branch is ignored.
-3. Wait for GitHub to report exactly `config::staging_checks` CI test
-   results for the staging branch. The project CI infrastructure is
-   expected to auto-test the staging branch on every change. A check
-   failure is a PR-specific step failure -- in GitHub terminology, all
-   staging checks are currently deemed "required". If discovered, any
-   extra check is considered a configuration problem (not related to any
```

Comment:
Now all 'extra' checks will be treated as optional (i.e., the checks that are not 'required').